cli/src/main/kotlin/DownloadCommand.kt

```diff
@@ -20,9 +20,12 @@
 package org.eclipse.apoapsis.ortserver.cli
 
 import com.github.ajalt.clikt.command.SuspendingNoOpCliktCommand
+import com.github.ajalt.clikt.core.Context
 import com.github.ajalt.clikt.core.subcommands
 
 class DownloadCommand : SuspendingNoOpCliktCommand(name = "download") {
+    override fun help(context: Context) = "Commands for downloading results and detailed information about ORT runs."
```

Comment:
FYI, I have a similar commit in #1873 already.

Reply:
Dropped the commit